packages/flutter_test/lib/src/matchers.dart

```diff
@@ -44,7 +44,7 @@ import 'widget_tester.dart' show WidgetTester;
 ///  * [findsAtLeast], when you want the finder to find at least a specific number of candidates.
 const Matcher findsNothing = _FindsCountMatcher(null, 0);
 
-/// Asserts that the [Finder] locates at least one widget in the widget tree.
+/// Asserts that the [FinderBase] locates at least one widget in the widget tree.
```

Comment:
I think this was correct as-is. findsWidgets deals with widgets and so does Finder.

Reply:
Ah ok you're right, they do all operate on FinderBase even when they have "widgets" in the name. I even see this below for findsOneWidget:

 > This is equivalent to the preferred [findsOne] method.

Indeed findsOne and findsOneWidget are identical.